.github/release-drafter.yml

```diff
@@ -15,29 +15,6 @@ categories:
           - 'style'
           - 'test'
 
```

Comment:
Can I ask u to fix these indentations to match the one on `cryostatio/cryostat`?